rest/route/distro.go

```diff
@@ -89,12 +90,17 @@ func (dgh *distroGetHandler) Run(ctx context.Context) gimlet.Responder {
 		return gimlet.MakeJSONErrorResponder(err)
 	}
 
+	isSuper := auth.IsSuperUser(dgh.sc.GetSuperUsers(), gimlet.GetUser(ctx))
```

Comment:
Instead of doing this check, why not make the entire route superuser only? Is there a use case for a non-superuser wanting to read a subset of fields from a distro?